app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RegistrationsController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)
+
+    if @user.save
+      redirect_to login_url, alert: t('.verify_email')
+    else
+      render :new, status: :unprocessable_entity
```

Comment:
show all the errors here. all the errors should be listed in list format. Refer screenshot
<img width="973" alt="Screenshot 2023-05-15 at 1 57 46 PM" src="https://github.com/rishabh-vinsol/restaurant-pos/assets/31533848/67e2193a-43e9-4f00-9615-7298fdde8647">